robot/client/client.go

```diff
@@ -886,6 +905,8 @@ func (rc *RobotClient) Status(ctx context.Context, resourceNames []resource.Name
 }
 
 // StopAll cancels all current and outstanding operations for the robot and stops all actuators and movement.
+//
+//	err := robot.StopAll(ctx.Background())
```

Comment:
```suggestion
//	err := machine.StopAll(ctx.Background())
```
I'm on the fence as to whether `client` might actually be a better name for the variable, but we should be consistent in this file. Ultimately the object is a client with a connection to the machine, not the machine itself. Kinda goes for all the documentation in the file as well, whether we should be more explicit about the relationship of the client to the machine or simplify it

Reply:
Changing to client would have to be done in the app code samples and on all the code sample throughout the docs that we've been adding. I think thinking of the client as the machine is probably more useful so would keep as is